cli-tools/dependencies-tree.txt

```diff
@@ -0,0 +1,1439 @@
+
```

Comment:
what's the use for these files?

Reply:
Mostly as a way to track down _why_ a dependency was included in the lock file -- as the default gradle lock files don't include this information.